File: wildfire-api/src/main/java/com/project/wildfire/Models/Cinder.java
package com.project.wildfire.Models;

import com.project.wildfire.Models.Abstract.AbstractCell;

import lombok.*;

/**
 * Class for cinder
 */
@Data
@EqualsAndHashCode(callSuper = true)
@ToString(callSuper = true)
public class Cinder extends AbstractCell {

    /**
     * Default constructor
     */
    public Cinder() {
        super();
    }

    /**
     * Builder used to create cinder cell with position
     * @param pos Position
     */
    @Builder
    public Cinder(Position pos){
        super(pos);
    }
}

File: wildfire-front/src/app/models/cinder.ts
import {Cell} from "./abstract/cell";
import {CellStatus} from "./enums/cell-status";

/**
 * Model for cinder
 */
export class Cinder extends Cell{

  /**
   * Constructor
   */
  constructor() {
    super(CellStatus.CINDER);
  }
}
